admin/routes/school.js

```diff
@@ -15,6 +15,7 @@ const { getHome,
   savePupilNotTakingCheck,
   removePupilNotTakingCheck,
   viewPupilsNotTakingTheCheck } = require('../controllers/school')
+const downloadFile = require('../controllers/downloadFile')
```

Comment:
shouldn't the controller name be download-file to follow the naming convention we are using for the rest of the controller names?
Also the controller's name is to generic to me. 
For example if you are building routes for the related panel why not call it related?

Reply:
@GeorgeChatzigiannis The routes are download-guidance for each of the three roles. The controller will grow soon (please see previous reply) to cover other downloads, hence the generic name.
I have modified the controller's file name to lower chase and hyphen as you suggested.

Reply:
if it was generic you would need to be very careful about how you pass details to the controller about what to download.  It may just be safer, and easier to maintain if the controller just serves the one file.  the current name is fine until you introduce another.